app/Filament/Resources/EventResource.php

```diff
@@ -105,6 +106,8 @@ public static function infolist(Infolist $infolist): Infolist
 
     public static function form(Form $form): Form
     {
+        $isNew = !is_a($form->model, Event::class);
```

Comment:
https://stackoverflow.com/questions/3017684/what-is-the-difference-between-is-a-and-instanceof